### PR TITLE
[do-not-merge][full-ci][tests-only]Debug playwright trace 

### PR DESCRIPTION
**Pull request by amrita-shrestha**
## Description
This PR is just for testing playwright trace.

### CODE REVIEW

File: tests/e2e/cucumber/environment/index.ts
Comment (saw-jan) [inline tests/e2e/cucumber/environment/index.ts:1]: I would recommend to just revert the config change of PR https://github.com/owncloud/web/pull/11640/files#diff-9f9314400936aaefcd87a9cb13424277040942a31ef7338ecda1df5278ce7276R145-R148

and check that if tracing dir exists before performing anything in function `filterTracingReports`. Tha should fix both issues

Comment (amrita-shrestha) [reply tests/e2e/cucumber/environment/index.ts:1]: i already found the reason but not clear what you want to achieve on that PR

Comment (saw-jan) [reply tests/e2e/cucumber/environment/index.ts:1]: I tried to fix the following problem:
```
    ✖ failed
      Error: ENOENT: no such file or directory, scandir 'reports/e2e/playwright/tracing'
          at Object.readdirSync (node:fs:1507:26)
          at filterTracingReports (file:///var/www/owncloud/web/tests/e2e/cucumber/environment/index.ts:135:28)
          at World.<anonymous> (file:///var/www/owncloud/web/tests/e2e/cucumber/environment/index.ts:112:9)
          at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```
Let's check the tracing dir if exists in `filterTracingReports`, that should fix both problems

**Comment by amrita-shrestha** [comment]
purpose has been served so closing this PR code will be merge on https://github.com/owncloud/web/pull/11659

**Pull request closed without merging**